tests/testutil/service.go

```diff
@@ -152,6 +139,20 @@ func (ts *TestService) ExportRaw() TestService {
 // Getters
 //////////
 
+// GetEndpoint performs an http.get against the service endpoint and returns response if
+// status code is 200, otherwise it errors out
+func (ts *TestService) GetEndpoint() string {
+	endpoint, err := ts.waitForEndpointDNS()
```

Comment:
I like how below in `GetAppEndpointResponse` you reuse the `GetAppEndpoint` method to get the url. Lets follow that pattern here and use `GetEndpointURL` to get the url and pass that into the Wait method.